packages/providers/src/transaction-response/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { arrayify } from '@ethersproject/bytes';
```

Comment:
should this file be removed now that `processGqlReceipt` is inside of `transaction-summary?`